lib/wise_homex/api_client_impl/api_client_impl.ex

```diff
@@ -375,6 +375,17 @@ defmodule WiseHomex.ApiClientImpl do
     Request.patch(config, "/properties/#{property_id}/import-relationships", payload)
   end
 
+  def property_reset(config, property_id) do
```

Comment:
I think that implementation in `lib/wise_homex/test/api_client_mock/api_client_mock.ex` is also needed

Additionally IMO it would be nice to have this request tested as it is here: https://github.com/wise-home/wise_homex/blob/master/test/api_client_test.exs#L183

Reply:
Added tests